packages/quip-cli/test/test-util.ts

```diff
@@ -22,11 +22,11 @@ export const useFixtureDir = (dir: string) => {
         return cleanFixtures(true);
     };
 };
-export const readManifestContent = async (dir?: string): Promise<string> => {
+export const readManifestContent = (dir?: string): string => {
     const mPath = dir ? path.join(dir, "manifest.json") : "manifest.json";
-    return String(await fs.promises.readFile(mPath, "utf-8"));
+    return String(fs.readFileSync(mPath, "utf-8"));
```

Comment:
good question! i was running tests via `npm test` locally and noticed that the `quip-cli` was flaky - particularly around the parts where it would read in pre-created files to compare with the command outputs. my theory is that the test times out before it can completely (and consistently) read the files. by forcing it to be synchronous the flakiness disappeared. i ran it several times consecutively and saw 0 failures

Reply:
Makes sense! Thanks for the fix!